From d76b3da47ed014f41ec5011d72de330fc7b35d06 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Wed, 7 Dec 2022 11:24:55 +0100
Subject: [PATCH] Do not disable omnia image

OpenWrt uses Linux 5.10 which has an issue with the Omnia switch, but we have
5.15, so this doesn't apply to us.

disabled in https://github.com/openwrt/openwrt/commit/a0bae2fef8bdd4f76767e1b29deb1adf279403e9
---
 target/linux/mvebu/image/cortexa9.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/linux/mvebu/image/cortexa9.mk b/target/linux/mvebu/image/cortexa9.mk
index 102e9a0f38..5d8380e464 100644
--- a/target/linux/mvebu/image/cortexa9.mk
+++ b/target/linux/mvebu/image/cortexa9.mk
@@ -62,7 +62,6 @@ define Device/cznic_turris-omnia
     partx-utils kmod-i2c-mux-pca954x kmod-leds-turris-omnia
   SUPPORTED_DEVICES += armada-385-turris-omnia
   BOOT_SCRIPT := turris-omnia
-  DEFAULT := n
 endef
 TARGET_DEVICES += cznic_turris-omnia
 
-- 
2.38.1

